tests/example/rewrite/rewrite.dl

```diff
@@ -0,0 +1,73 @@
+// Souffle - A Datalog Compiler
+// Copyright (c) 2020, The Souffle Developers. All rights reserved
+// Licensed under the Universal Permissive License v 1.0 as shown at:
+// - https://opensource.org/licenses/UPL
+// - <souffle root>/licenses/SOUFFLE-UPL.txt
+
```

Comment:
A brief note explaining what this example is for would be great to have here. That and removing the spaces from end of lines.